src/MovieCard/MovieCard.js

```diff
@@ -13,35 +12,32 @@ class MovieCard extends Component {
       error: null,
       deleted: false,
 		};
+		this.ratingButtons = this.createRadioButtons();
 	}
```

Comment:
instead of createRadioButtons being called every time MovieCard is re-rendered, like this it will be set on the creation of the Component. This functionality doesn't depend on any other variable since it will always be 10 radio buttons rendered.

Reply:
It seems so small but why not just create them once and save to a variable. Could probably be very useful if you made something very complicated